daangn.css

```diff
@@ -0,0 +1,161 @@
+/* footer body */
+#footer_root {
+  border-top: 1px solid lightgray;
+  padding: 0 1.6rem;
+  box-sizing: border-box;
+  width: 100%;
+  max-width: 100vw;
+}
+.footer_body {
+  display: flex;
+  flex-direction: column;
+  max-width: 50rem;
+  margin: 0 auto;
+}
+
+/* footer section 1 */
+.footer_section1 {
+  padding: 4.8rem 0;
+  display: block;
+}
+.footer_menus {
+  display: flex;
+  max-width: 76.8rem;
+  font-size: 0.8rem;
+}
+.footer_menus_row {
+  display: flex;
+  flex-direction: column;
+  padding-inline-start: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  width: 33%;
+  list-style-type: disc;
+}
+.footer_menus_item:not(:last-child) {
+  margin-bottom: 1.6rem;
+}
+.footer_menus_item {
+  width: fit-content;
+  list-style: none;
+  margin: 1.6rem;
+}
+.footer_menus_item a {
+  text-decoration: none;
+  outline: 0;
+  color: black;
+}
+
+.footer_download {
+  float: right;
+  margin-bottom: 0;
+  display: flex;
+  flex-direction: column;
+}
+.footer_download_text {
+  line-height: 1.4;
+  font-size: 0.8rem;
+  font-weight: 700;
+}
+.footer_download_buttons {
+  display: flex;
+}
+.footer_download_button {
+  display: block;
+  margin-top: 0.8rem;
+  text-decoration: none;
+  outline: 0;
+  display: flex;
+  color: black;
+  font-weight: 700;
+  font-size: 0.9rem;
+  padding: 0.2rem;
+}
+.ml {
+  margin-left: 1rem;
+}
+.footer_download_button div {
+  width: 100%;
+  padding: 10px;
+  border-radius: 10px;
+  background-color: lightgray;
```

Comment:
원본보다 약간 어두운 것 같아서 색상을 #eaebee로 바꿔도 좋을 것 같아요 !
```suggestion
  background-color: #eaebee
```
<img width="30" alt="스크린샷 2024-03-19 오전 11 06 05" src="https://github.com/rokiki2/Daangn_Clone/assets/103100783/a94e0e41-2599-41c2-b3bf-ed3f68076de4">
개발자도구에서 해당 툴을 누른 다음 원하는 요소를 클릭하면 해당 요소에 적용된 css도 확인이 가능하니 참고해 주세요 !!